### PR TITLE
Add support for mod operation on PK

### PR DESCRIPTION
**Pull request by AlexeySoshin**
Possible solution for #1597

### CODE REVIEW

**Comment by Tapac** [comment]
@AlexeySoshin , could you please also add `mod` functions similar to `rem`?

**Comment by AlexeySoshin** [comment]
@Tapac - added. Hope I understood you correctly.

File: detekt.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # https://github.com/detekt/detekt/blob/main/detekt-core/src/main/resources/default-detekt-config.yml
 
 build:
-    maxIssues: 52
+    maxIssues: 200
```
Comment (AlexeySoshin) [inline detekt.yml:4]: @Tapac 
I raised this as the current error count is 143, and I think Detekt errors should be addressed in a different PR

Comment (Tapac) [reply detekt.yml:4]: May I ask you to merge master into the branch? Detekt issue was solved there.
